presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -132,7 +134,8 @@ public static HiveMetastore cachingHiveMetastore(HiveMetastore delegate, Executo
                         .map(Duration::toMillis)
                         .map(OptionalLong::of)
                         .orElseGet(OptionalLong::empty),
-                maximumSize);
+                maximumSize,
+                true);
```

Comment:
Please define an enum. That way it will be much more readable when you pass `STATS_RECORDING_ENABLED` (or `StatsRecording.ENABLED`) instead of `true`.

Reply:
Done